params/astria_config.go

```diff
@@ -0,0 +1,444 @@
+package params
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"math/big"
+	"sort"
+
+	"github.com/btcsuite/btcd/btcutil/bech32"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/crypto"
+	"github.com/ethereum/go-ethereum/log"
+)
+
+type AstriaForks struct {
+	orderedForks []AstriaForkData // sorted in descending order by Height
+	forkMap      map[string]AstriaForkConfig
+}
+
+type AstriaForkConfig struct {
+	Height            uint64                            `json:"height"`
+	Halt              bool                              `json:"halt,omitempty"`
+	SnapshotChecksum  string                            `json:"snapshotChecksum,omitempty"`
+	ExtraDataOverride hexutil.Bytes                     `json:"extraDataOverride,omitempty"`
+	FeeCollector      *common.Address                   `json:"feeCollector,omitempty"`
+	EIP1559Params     *AstriaEIP1559Params              `json:"eip1559Params,omitempty"`
+	Sequencer         *AstriaSequencerConfig            `json:"sequencer,omitempty"`
+	Celestia          *AstriaCelestiaConfig             `json:"celestia,omitempty"`
+	BridgeAddresses   []AstriaBridgeAddressConfig       `json:"bridgeAddresses,omitempty"`
```

Comment:
trying to wrap my head if we will need a `startHeight` for `AstriaBridgeAddressConfig` and `EIP1559Params` after adding named forks. I can understand the requirement for `startHeight` in sequencer and celestia heights since i believe they refer to the start heights of the sequencer and celestia chain at which syncing starts happening.

Reply:
you're thinking is correct, we don't need a start height for `AstriaBridgeAddressConfig` or `EIP1559Params` as their start height is a rollup height inherited from the fork height. Sequencer and celestia heights are heights for those chains